src/commands/publish-local.js

```diff
@@ -2,11 +2,10 @@ import colors from 'colors';
 import path from 'path';
 import { paramCase } from 'change-case';
 import settle from 'promise-settle';
-import mkdirp from 'mkdirp';
 import fs from 'fs';
 
 import { generateSpecsArchive } from '../lib/archiver';
-import { getTempDir, getPOMContent } from '../lib/utils';
+import { getPOMContent, getTempDir } from '../lib/utils';
```

Comment:
Why this change?

Reply:
Auto formatting i guess, no real necessity